plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestDefaultJdbcQueryBuilder.java

```diff
@@ -574,7 +573,7 @@ public void testEmptyBuildSql()
                     "FROM \"test_table\" " +
                     "WHERE \"col_1\" IS NULL");
             try (ResultSet resultSet = preparedStatement.executeQuery()) {
-                assertEquals(resultSet.next(), false);
+                assertThat(resultSet.next()).isEqualTo(false);
```

Comment:
There may be an IDEA inspection for that:

```suggestion
                assertThat(resultSet.next()).isFalse();
```